shared/chat/conversation/messages/wrapper/unfurl/generic/index.stories.tsx

```diff
@@ -1,15 +1,16 @@
 import * as React from 'react'
-import {Box} from '../../../../../../common-adapters/index'
+import {Box} from '../../../../../../common-adapters'
+import * as Styles from '../../../../../../styles'
 import * as Sb from '../../../../../../stories/storybook'
 
 import UnfurlGeneric from '.'
 
 const full = {
   description:
     'A surge in technology shares following Facebook’s latest earnings lifted U.S. stocks, helping major indexes trim some of their October declines following a punishing period for global investors.',
-  faviconURL: require('../../../../../../images/mock/wsj.jpg'),
+  faviconURL: Styles.isMobile ? '' : require('../../../../../../images/mock/wsj.jpg'),
```

Comment:
Why not show it on mobile? Should it be checking for storyshot instead?

Reply:
sure